buildSrc/src/main/kotlin/multiplatform-common.gradle.kts

```diff
@@ -0,0 +1,78 @@
+@file:Suppress("UNUSED_VARIABLE")
+
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.dokka")
+    `maven-publish`
+}
+
+repositories {
+    mavenCentral()
+
+    jcenter()
+}
+
+kotlin {
+    jvm()
+    js()
+    linuxX64()
+    macosX64()
+    mingwX86()
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(kotlin("stdlib-common"))
+            }
+        }
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test-common"))
+                implementation(kotlin("test-annotations-common"))
+            }
+        }
+        val jvmMain by getting {
+            dependencies {
+                implementation(kotlin("stdlib"))
+            }
+        }
+        val jvmTest by getting {
+            dependencies {
+                implementation(kotlin("test-junit"))
+            }
+        }
+        val jsMain by getting {
+            dependencies {
+                implementation(kotlin("stdlib-js"))
+            }
+        }
+        val jsTest by getting {
+            dependencies {
+                implementation(kotlin("test-js"))
+            }
+        }
+
+        val nativeMain = create("nativeMain") {
+            dependsOn(commonMain)
```

Comment:
Don't you want an `stdlib` target for this?

Reply:
commonMain has std-lib, there's not an explicit one for native stuff. This is just making a build type that can be shared between all native builds.

Reply:
That's weird that all the others have one but not native 🤔